bitcointranscripts/main.py

```diff
@@ -98,25 +98,26 @@ def parse_post(file_path):
     front_matter, body = parse_markdown(content)
     # Extract metadata from front matter using yaml
     metadata = yaml.safe_load(front_matter)
-    url_path = file_path.replace('.md', '').replace(GLOBAL_URL_VARIABLE, '')
+    url_path = os.path.normpath(file_path.replace('.md', '').replace(GLOBAL_URL_VARIABLE, ''))
     transcript_by = metadata.get('transcript_by', "")
 
     document = {
-        'id': "bitcointranscripts" + url_path.replace("\\", "+").replace("/", "+"),
+        'id': "bitcointranscripts" + url_path.replace(os.sep, "+"),
         'title': metadata['title'],
         'body_formatted': body,
         'body': body,
         'body_type': "markdown",
         'created_at': (datetime.strptime(metadata['date'], '%Y-%m-%d').strftime('%Y-%m-%dT%H:%M:%S.000Z') if isinstance(metadata.get('date'), str) else None),
         'domain': "https://btctranscripts.com/",
-        'url': "https://btctranscripts.com" + url_path.replace("\\", "/"),
+        'url': "https://btctranscripts.com" + url_path.replace(os.sep, "/"),
         'categories': metadata.get('categories', []),
         'tags': metadata.get('tags', []),
         'media': metadata.get('media', ""),
         'authors': metadata.get('speakers', []),
         'indexed_at': datetime.now().isoformat(),
         'transcript_by': transcript_by,
-        'needs_review': "Yes" if "--needs-review" in transcript_by else "No"
+        'needs_review': "Yes" if "--needs-review" in transcript_by else "No",
```

Comment:
This should be a comment on your previous PR, but anyway: 
elasticsearch supports boolean, is there a benefit of having this as text? I guess boolean will make it easier to query for this?

Reply:
Right, Let's switch it to Boolean. Boolean will help in efficient storage and faster queries.